cmd/gctcli/commands.go

```diff
@@ -3529,8 +3271,8 @@ func getAuditEvent(c *cli.Context) error {
 
 var uuid, filename, path string
 var gctScriptCommand = cli.Command{
-	Name:      "gctscript",
-	Usage:     "execute gctscript command",
+	Name:      "script",
```

Comment:
any reason this was renamed?

Reply:
just a slight stutter and gctcli.exe gctscript ... and expanded the usage to make it easier for newbies, if you want it reverted, happy too.

Reply:
happy to default it to script and have gctscript as an alias 

Also regarding the usage part the only issue i had was with the wording "virtual machine scripting management" it makes it seem like its an isolated environment when really its executed directly within the bot and can control aspects of it 

(i understand its refered to as VM in the code base looking back this is one thing i would have changed :D)

Reply:
so revert it back to execute gctscript command?

Reply:
happy to drop the gct part and just have it as "execute script" if you would rather not prefix by "gct"